ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -4,21 +4,28 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "RoleName": "hmcts-admin",
     "ReadOnly": "Y",
-    "AccessProfiles": "GS_profile"
+    "AccessProfiles": "GS_profile,caselinks_admin_profile"
```

Comment:
as mentioned in standup - just need to confirm these hmcts-roles having this level of access, as they're read only roles granted to all internal users, not just publiclaw ones